dirty_cat/test/test_string_distances.py

```diff
@@ -22,6 +23,19 @@ def _random_string_pairs(n_pairs=50):
         pairs.append((s1, s2))
     return pairs
 
+def _random_common_char_pairs(n_pairs=50):
+    """
+    Return string pairs with a common char at random positions. This should
+    discriminate different thresholds for matching chararacters in Jaro distance.
+    """
```

Comment:
The code in this function does not follow the PEP8 coding convention: you need spaces after the commas, and before and after "+", "*".